plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -252,6 +255,10 @@ export function patchCallback(
     res: object
   ) {
     if (err) {
+      if (Object.keys(err).includes('code')) {
```

Comment:
nit: maybe is not necessary to create an array of keys to check the presence of `code`. I think you could make use of `Object.hasOwnProperty`